addons/project/data/mail_message_subtype_data.xml

```diff
@@ -59,7 +59,7 @@
         <field name="hidden" eval="True"/>
     </record>
     <record id="mt_task_dependency_change" model="mail.message.subtype">
-        <field name="name">Task Dependency Changes</field>
+        <field name="name">Task Dependencies Done</field>
```

Comment:
Why is it also `Task Dependencies Done`? 🤔

Reply:
I'm not sure, but i think its because in one case its shown on the project chat, on the other case its shown on the task, but its always a notification about the task depencies done. As far as i know, there's no way to make a project dependent on another, is there ? :thinking:

Reply:
The feature only exists in tasks and so a task could depend on another task in another project for instance. 🙂 
So, I am not sure to understand, I will check the code. 🤔

Reply:
Why "done" though? This is used to log changes of a tracked field in a child task afaik
It doesn't necessarily mean it's done, right?

Reply:
This was updated to be consistent with the terminology used with the new 'mark as done' system of the tasks. The task dependency change will be removed in a following task anyway, since the 'mark as done' introduced its own system.

Reply:
Since those subtypes will be removed in another task, we will remove those changes. 🙂